src/hashmap.c

```diff
@@ -137,7 +137,7 @@
     kh_clear(T##_##U##_map, map->table);                                      \
   }
 
-MAP_IMPL(string, ptr_t, DEFAULT_INITIALIZER)
+MAP_IMPL(string, ptr_t, DEFAULT_INITIALIZER) /* maps pluginkey <> connection */
 MAP_IMPL(cstr_t, ptr_t, DEFAULT_INITIALIZER)
 MAP_IMPL(string, dispatch_info, {.func = NULL, .async = false, .name = {.str = NULL, .length = 0}})
-MAP_IMPL(uint64_t, string, {.str = NULL, .length = 0})
+MAP_IMPL(uint64_t, string, DEFAULT_INITIALIZER) /* maps callid <> pluginkey */
```

Comment:
Good find! This is legacy code because I forgot to change the initializer after I changed pluginkey to `string`. Fixed in 0048859bd7927d308bf004a295f0a6b3e2d95bf2.